app/design/frontend/Diana/default-theme/web/js/render-stores.js

```diff
@@ -0,0 +1,114 @@
+define([
+    'uiComponent',
+    'jquery',
+    'ko',
+    'underscore',
+    'js/selector-store',
+    'js/selector-city'
+], function(Component, $, ko, _){
+    'use strict';
+
+    return Component.extend({
+        defaults: {
+            currentPageNumber: ko.observable(1),
+            elementsPerPage: 6,
+            quantityOfPages: ko.observable(),
+            quantityOfPagesRenderer: ko.observableArray([]),
+            paginatedElements: ko.observableArray([]),
+            selectedCity: ko.observable(),
+            paramPage: ko.observable(),
+            imports: {
+                // Observable variable of selected city by user
+                selectedCity: 'selectorCity:selectedCity',
+                // Observable array with all available stores filtered with selected city and brand
+                availableStores: 'selectorStore:availableStores'
+            }
+        },
+        initObservable: function () {
+            this._super().observe(['selectedCity', 'availableStores']);
+
+            return this;
+        },
+        initialize: function() {
```

Comment:
@Dianaeva Please, add empty lines between methods. Also it is recommended to add comment with description of functionality for methods for better maintenance